### PR TITLE
Update 3.1 SiteExtension version

### PR DESCRIPTION
**Pull request by wtgodbe**

### CODE REVIEW

File: eng/Versions.props [hunk 1]
```diff
@@ -210,7 +210,7 @@
...
     <MicrosoftAspNetCoreAzureAppServicesSiteExtension21PackageVersion>2.1.1</MicrosoftAspNetCoreAzureAppServicesSiteExtension21PackageVersion>
     <MicrosoftAspNetCoreAzureAppServicesSiteExtension22PackageVersion>2.2.0</MicrosoftAspNetCoreAzureAppServicesSiteExtension22PackageVersion>
-    <MicrosoftAspNetCoreAzureAppServicesSiteExtension31PackageVersion>3.1.9-servicing-20473-9</MicrosoftAspNetCoreAzureAppServicesSiteExtension31PackageVersion>
+    <MicrosoftAspNetCoreAzureAppServicesSiteExtension31PackageVersion>3.1.10-servicing-20520-4</MicrosoftAspNetCoreAzureAppServicesSiteExtension31PackageVersion>
     <MicrosoftAspNetCoreAzureAppServicesSiteExtension31x64PackageVersion>$(MicrosoftAspNetCoreAzureAppServicesSiteExtension31PackageVersion)</MicrosoftAspNetCoreAzureAppServicesSiteExtension31x64PackageVersion>
     <MicrosoftAspNetCoreAzureAppServicesSiteExtension31x86PackageVersion>$(MicrosoftAspNetCoreAzureAppServicesSiteExtension31PackageVersion)</MicrosoftAspNetCoreAzureAppServicesSiteExtension31x86PackageVersion>
     <MicrosoftAspNetCoreAzureAppServicesSiteExtension50PackageVersion>5.0.0-preview-7-20365-19</MicrosoftAspNetCoreAzureAppServicesSiteExtension50PackageVersion>
```
Comment (dougbu) [inline eng/Versions.props:216]: This version should be `5.0.0-rtm-20529-7` now

**Review by dougbu — CHANGES_REQUESTED** [review]

**Comment by dougbu** [comment]
@BrennanConroy and @jkotalik will the site extension bundle from the release/5.0 branch also be used❔ Wondering if we need to bump to 3.1.10 there too.

File: eng/Versions.props (hunk 1 above)
Comment (wtgodbe) [reply eng/Versions.props]: done

**Comment by BrennanConroy** [comment]
Probably should, but it's not a big deal yet since I don't think any patches have been made to 3.1 with regards to its dependencies.

**Review by dougbu — APPROVED** [review]

**Comment by dougbu** [comment]
> Probably should

@wtgodbe can you take that please❔

**Comment by wtgodbe** [comment]
> @wtgodbe can you take that please❔

yep

**Comment by wtgodbe** [comment]
https://github.com/dotnet/aspnetcore/pull/27691

**Pull request merged**